addons/website_sale_picking/models/res_config_settings.py

```diff
@@ -0,0 +1,16 @@
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    picking_site_ids = fields.Many2many(
+        'delivery.carrier',
+        related='website_id.picking_site_ids',
+        readonly=False
+    )
+
+    default_picking_product_id = fields.Many2one('product.product',
+                                                 default_model='product.product',
```

Comment:
What is `default_model`? Does not seem to be an actual key.

Reply:
Tbh i'm not sure what that is, but I get an error if I don't set that attribute. I see it's used in other places as well.
(https://github.com/odoo-dev/enterprise/blob/6f2424f9a503024c812b8155be75f98b0e11eaab/l10n_be_hr_contract_salary/models/res_config_settings.py#L10)[HERE] is the only other place where I see that beeing used. I'll remove it if the error is gone